tests/functional/test_mlops_monitoring.py

```diff
@@ -240,37 +236,55 @@ def test_drum_regression_model_monitoring_fails_in_unstructured_mode(
         "framework, problem, language",
         [(None, UNSTRUCTURED, PYTHON_UNSTRUCTURED_MLOPS)],
     )
-    @pytest.mark.parametrize(
-        "with_monitor_settings",
-        [False, True],
-    )
-    def test_drum_unstructured_model_embedded_monitoring(
-        self, resources, framework, problem, language, tmp_path, with_monitor_settings
+    def test_drum_unstructured_model_embedded_mlops_reporting(
```

Comment:
I suggest keeping this. It can verify that overriding with env vars works as expected.

Reply:
There's another dedicated test for the monitor settings (see `test_drum_unstructured_model_with_stdout_spooler_type` below).
This test case validates that reporting access are actually take place. The monitor settings are no longer required for that.